### ISSUE TITLE
TextField from this package overrides global variant prop

### ISSUE THREAD

**Issue by JReinhold** [stackworx/formik-material-ui#87]
# The Problem

In Material-UI, it is possible to set global default `props` ([see here](https://material-ui.com/customization/globals/#default-props)).

However the `TextField` component in this package defaults to always setting the `variant` prop to `standard`, which effectively overrides any default values set for this prop.

That means, that something like this is not possible:

```ts
import MuiThemeProvider from '@material-ui/core/styles/MuiThemeProvider';
import createMuiTheme from '@material-ui/core/styles/createMuiTheme';
import { TextField as FormikTextField } from 'formik-material-ui';
import MuiTextField from '@material-ui/core/TextField';
import { Field } from 'formik';

import * as React from 'react';

const theme = createMuiTheme({
  props: {
    MuiTextField: {
      variant: 'outlined'
    },
  }
});

export class App extends React.Component {
  public render() {
    return (
      <MuiThemeProvider theme={theme}>
        ...formik form
        <Field
          // will incorrectly be variant 'standard'
          component={FormikTextField}
        />
		// will correctly be variant 'outlined' from the theme
		<MuiTextField/>
      </MuiThemeProvider>
    );
  }
}
```

# The Cause

I believe that this issue comes from #85 (see [the offending line here](https://github.com/stackworx/formik-material-ui/pull/85/files#diff-4efea83650c353230d6e3e2f9d082685R13)). That PR doesn't have much info on it, so I can't really tell why this change was made.
Maybe @9renpoto can shed some light on that.

# Current workarounds

Pinning the version of this library to `v0.0.18`, the version before the mentioned pull request, fixes the issue. An alternative is to always pass the `variant` prop, and not rely on the default props feature that Material-UI has.

**Comment by cliedeman** [comment]
Silly of me to miss that. Will fix soon

**Comment by ajdaniel** [comment]
Just came here to raise this bug which took me 20 minutes to work out! Happy if I quickly raise a PR?

**Comment by cliedeman** [comment]
Sorry for the wait. Released in 0.22
